pkg/operator/sync_openshiftcontrollermanager_v311_00_test.go

```diff
@@ -201,3 +208,93 @@ func TestProgressingCondition(t *testing.T) {
 	}
 
 }
+
+func TestDeploymentWithProxy(t *testing.T) {
+	kubeClient := fake.NewSimpleClientset(
+		&appsv1.DaemonSet{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:       "controller-manager",
+				Namespace:  "openshift-controller-manager",
+				Generation: 2,
+			},
+		},
+	)
+	dsClient := kubeClient.AppsV1()
+	indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{})
+	proxyConfig := &configv1.Proxy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster",
+		},
+		Spec: configv1.ProxySpec{
+			NoProxy:    "https://no-proxy",
```

Comment:
nit - `NO_PROXY` is a comma-separated list of host names. For a unit test this doesn't really matter.

Reply:
fixed this too